testsuite/features/build_validation/core/allcli_sanity.feature

```diff
@@ -25,6 +25,7 @@ Feature: Sanity checks
     And socket "tftp" is active on "server"
     And service "tomcat" is enabled on "server"
     And service "tomcat" is active on "server"
+    And files on container volumes should all have the proper SELinux label
```

Comment:
I guess it right that this label must be there not only if we are under a transaction-system, but also a regular one?
I'm thinking in our AWS deployment where we deploy our server on top of Leap, or the idea of supporting non-transactional systems as Host OS in the future versions

Reply:
The labels are on any system with SELinux.

But the container volumes are on containerized systems only.

From what I understood, the problems emerge only when we are combining SELinux and container volumes, https://blog.christophersmart.com/2021/01/31/podman-volumes-and-selinux/ .

The transactional aspect does not play a role AFAIK. It just happens to be a coincidence that those systems are all shipped with SELinux.

For your question, it depends whether you are deployer with containers (5.0 or 5.1) or without (4.3) in the cloud.
I hope I answered it?